File: frontend/src/app/features/connections/models/connection-response.ts
import { BaseModel } from '../../../core/models/base-model';

export interface ConnectionResponse extends BaseModel {
  clientGaia: string;
  appId: string;
  processId: string;
  apiVersion: string;
  machine: string;
  establishedAt: string;
  terminatedAt?: string;
}

File: http-client/MCS.WatchTower.WebApi.DataTransferObjects/Responses/ConnectionResponse.cs
namespace MCS.WatchTower.WebApi.DataTransferObjects.Responses;

public record ConnectionResponse : IBaseDto
{
    public string Id { get; init; }
    public string ClientGaia { get; init; }
    public string AppId { get; init; }
    public string ApiVersion { get; init; }
    public string Machine { get; init; }
    public string ProcessId { get; init; }
    public DateTime EstablishedAt { get; init; }
    public DateTime? TerminatedAt { get; init; }
}
